png.go

```diff
@@ -94,5 +95,12 @@ func resvgConvert(doc *etree.Document, w, h float64, output string) error {
 	if err != nil {
 		return err
 	}
+	if copy {
+		err = clipboard.Init()
+		if err != nil {
+			return err
+		}
+		clipboard.Write(clipboard.FmtImage, png)
+	}
```

Comment:
I am going to add `clipboard.Read(clipboard.FmtImage)` to see if it works that way for @Moulick.